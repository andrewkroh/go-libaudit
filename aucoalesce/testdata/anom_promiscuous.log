type=ANOM_PROMISCUOUS msg=audit(1492734742.981:753): dev=ens4 prom=0 old_prom=256 auid=1001 uid=0 gid=0 ses=1
type=SYSCALL msg=audit(1492734742.981:753): arch=c000003e syscall=16 success=yes exit=0 a0=4 a1=8914 a2=7ffdaff944b0 a3=0 items=0 ppid=1852 pid=1926 auid=1001 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=pts0 ses=1 comm="ifconfig" exe="/sbin/ifconfig" key=(null)
type=PROCTITLE msg=audit(1492734742.981:753): proctitle=6966636F6E66696700656E7334002D70726F6D697363
type=EOE msg=audit(1492734742.981:753):