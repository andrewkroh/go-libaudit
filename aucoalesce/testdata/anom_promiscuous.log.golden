{
  "@timestamp": "2017-04-21T00:32:22.981Z",
  "sequence": 753,
  "category": "anomoly",
  "record_type": "anom_promiscuous",
  "result": "success",
  "session": "1",
  "actor": {
    "primary": "1001",
    "secondary": "0",
    "attrs": {
      "egid": "0",
      "euid": "0",
      "fsgid": "0",
      "fsuid": "0",
      "gid": "0",
      "sgid": "0",
      "suid": "0"
    }
  },
  "thing": {
    "primary": "ens4"
  },
  "action": "promiscuous-mode-on-device",
  "data": {
    "a0": "4",
    "a1": "8914",
    "a2": "7ffdaff944b0",
    "a3": "0",
    "arch": "x86_64",
    "auid": "1001",
    "comm": "ifconfig",
    "dev": "ens4",
    "exe": "/sbin/ifconfig",
    "exit": "0",
    "gid": "0",
    "old_prom": "256",
    "pid": "1926",
    "ppid": "1852",
    "proctitle": "ifconfig",
    "prom": "0",
    "ses": "1",
    "syscall": "ioctl",
    "tty": "pts0",
    "uid": "0"
  }
}
